hq/app/model/models.scala

```diff
@@ -271,7 +277,28 @@ case class IAMAlertTargetGroup(
   users: Seq[VulnerableUser]
 )
 
-case class VulnerableUser(username: String, tags: List[Tag], disableDeadline: Option[DateTime] = None) extends IAMAlert
+case class VulnerableUser(
+  username: String,
+  key1: AccessKey = AccessKey(NoKey, None),
+  key2: AccessKey = AccessKey(NoKey, None),
+  humanUser: Boolean,
+  tags: List[Tag],
+  disableDeadline: Option[DateTime] = None
+) extends IAMAlert
+
+case class VulnerableAccessKey(
+  username: String,
+  accessKeyWithId: AccessKeyWithId,
+  humanUser: Boolean
+)
+object VulnerableAccessKey {
+  def isOutdated(user: VulnerableAccessKey): Boolean = {
```

Comment:
I think this would sit more naturally as a method in the class, rather than a companion object